plan/main

```diff
@@ -1,11 +1,11 @@
 #include <turnkey/base>
-#include <turnkey/lamp>
+#include <turnkey/lamp81>
```

Comment:
Did you forget to do a `common` PR for this new plan? It'd be easy for me to create one myself, but if you still have it handy, could you please open a new PR for this? Cheers man.

Reply:
Spot on, I've added this in common, along with dynamic pins, see below.